vuepress/docs/next/tutorials/create/pb/export-bundle-from-application.md

```diff
@@ -4,66 +4,61 @@ sidebarDepth: 2
 # Export and Publish a Bundle
 
 ## Overview
-Use the Entando bundler command to export a bundle of Entando components from an existing Entando application. An Entando bundle can be used to do the initial install of Entando components into an Entando application, migrate Entando components from one environment to another (e.g. Dev to QA), to provide a template for building a new Entando application, or as the skeleton of an Entando solution. The output of this command is the same bundle directory structure created by an Entando project including a bundle descriptor file.
+Use the bundler command to export a bundle of components from an existing Entando Application. An Entando Bundle can be used 
+- For the initial install of Entando components into an Entando Application
+- To migrate Entando components from one environment to another (e.g. Dev to QA)
+- To provide a template for building a new Entando Application
+- As the skeleton of an Entando solution
+
+This command generates the same bundle directory structure as an Entando project, including a bundle descriptor file.
 
 ### Prerequisites
-* Use the [Entando CLI](../../../docs/reference/entando-cli.md#check-the-environment) to verify you have the prerequisites in place for this tutorial (e.g. git, entando-bundler). 
+* Use the [Entando CLI](../../../docs/reference/entando-cli.md#check-the-environment) to verify prerequisites (e.g. git, entando-bundler): 
 ``` sh
 ent check-env develop 
 ```
-* You'll also need a running Entando application.
-* You'll need admin access to Keycloak or admin access to Kubernetes to set it up.
+* A running Entando Application
+* [Admin access to Keycloak](../../../docs/consume/identity-management.md#logging-into-your-keycloak-instance)
 
-The tutorial assumes you're using an Entando quickstart application. You may need to adjust the specific URLs, credentials, namespaces, etc. for a custom application.
+This tutorial assumes you're using an Entando quickstart application. You may need to adjust specific URLs, credentials, namespaces, etc., for a custom application.
 
 ## Export an Entando Bundle
 
-### Setup the Keycloak client
-You'll need to setup a Keycloak client with the appropriate permissions for the bundler to access all of the necessary Entando APIs.
+### Set Up the Keycloak Client
+Configure a Keycloak client to grant the bundler access to the required Entando APIs.
 
-1. Find the Secret for the Keycloak admin account. If you already have the admin credentials, then you can skip to step 3. 
+1. Find the Secret for the Keycloak admin account. In a quickstart application, the Secret is named `default-sso-in-namespace-admin-secret`.
```

Comment:
We can drop step#1 as well. It was only needed to get the admin credentials. This has the steps starting with logging into Keycloak using the admin credentials...